webui/react/src/components/JupyterLabModal.tsx

```diff
@@ -266,8 +270,9 @@ const JupyterLabForm: React.FC<FormProps> = (
               placeholder="No template (optional)"
               value={fields.template}
               onChange={value => onChange?.({ key: 'template', value: value?.toString() })}>
-              {templates.map(temp =>
-                <Option key={temp.name} value={temp.name}>{temp.name}</Option>)}
+              {templates.map(temp => (
```

Comment:
nit: why the extra braces? is this an auto fix? there might be an eslint rule we could add. it might be nice to have for extra clarity.

Reply:
yeah this is the `react/jsx-wrap-multilines` eslint rule. There were a few JSX specific linting rules introduced as an attempt to standardize JSX so it's easier to read.